03-Custom-Actions-In-Python/03-Custom-Action-Features.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>Do we even need to explain this? Or alternately do we need to show that  user's <code>CustomActions</code></p><p>are themselves loggable? Here we are showing two ways to do the same thing. It would be best to just show the one preferred solution.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I think we need to show this pass through at least. We can make this section smaller.</p>